### ISSUE TITLE
Warning about interferences on the radio modules

### ISSUE THREAD

**Issue by RealZack** [home-assistant/home-assistant.github.io#21565]
### Feedback

Hello,

I'm using HA on a pi 4 and I'm using ZHA with a conbee 2.

I strungle a lot to get it working, it was impossible to do any connection, or if I was, I was unable to do anything with it.

I finally found out the problem checking other documentations such as the one from zigbee2mqtt or from phoscon and deconz.

There is known issues about getting 2.4GHZ dongle near usb 3.0 port.

I really think a disclamair about using an extension cable in the part **"BEST PRACTICES TO AVOID PAIRING/CONNECTION DIFFICULTIES"** could have save me a lot of time of debug/research

Hope it could help someone else

### URL

https://www.home-assistant.io/integrations/zha/

### Version

2022.2.5

**Comment by SNoof85** [comment]
See #18864 wich is still waiting for a review and has a warning about this feedback.

**Comment by Hedda** [comment]
FYI, help is wanted to make the text in https://github.com/home-assistant/home-assistant.io/pull/18864 more "readable"

**Comment by Hedda** [comment]
FYI, I have updated PR https://github.com/home-assistant/home-assistant.io/pull/18864 to try to make it "readable" enough to be accepted for merging. Waiting for feedback and reviews.

**Comment by Hedda** [comment]
Still waiting for new review of https://github.com/home-assistant/home-assistant.io/pull/18864 after made changes as requested.

**Comment by Hedda** [comment]
Not stale, still waiting for new review of https://github.com/home-assistant/home-assistant.io/pull/18864 after made changes as requested.

**Comment by Hedda** [comment]
Still not stale, just missing reviewers.

**Comment by Hedda** [comment]
Giving this another try with a fresh new pull request for ZHA docs -> https://github.com/home-assistant/home-assistant.io/pull/24432

**Comment by nawwman** [comment]
Agree this would be helpful to add to documentation

**Comment by Hedda** [comment]
@frenck Can you please unlock the https://github.com/home-assistant/home-assistant.io/pull/18864 pull request now?

You did reopen https://github.com/home-assistant/home-assistant.io/pull/18864 but it is locked so cannot comment on it and assume no one else can comment to review either it either. 

Anyway, I did commit a rewrite to the patch branch so at least that is now in PR https://github.com/home-assistant/home-assistant.io/pull/18864 (as in https://github.com/home-assistant/home-assistant.io/pull/24432) -> https://github.com/home-assistant/home-assistant.io/pull/18864/commits/bc2e917c947f393df6bfb20f6e4c3b5ce6f20a9e

So now the new section reads very similarly to the Bluetooth integration documentation with exception of the relevant differences for Zigbee network mesh ->  https://www.home-assistant.io/integrations/bluetooth/#bluetooth-interference-with-other-devices

### Zigbee interference avoidance and network range/coverage optimization

Sources of interference for radios can lead to transmission/reception loss or connection problems and show symptoms such as errors/failures when sending and receiving Zigbee messages/signals that can cause significant degradation in performance or even prevent devices from communicating at all. Below are some basic but essential tips for getting a good setup starting point to achieve better signal quality, improved coverage, and extended range.

Following all these optimization tips below should significantly improve the reception of your Zigbee radio adapter. The below insights describe working around the well-known limitations of low-power/low-bandwidth 2.4 GHz digital radios. It can that way resolve or avoid many known issues caused by interference or poor placement of your Zigbee radio adapter or devices.

All electric devices/appliances, especially computers and computer peripherals, generate [electromagnetic interference (also known as EMI/EMI/RMI, or signal noise in layman's terms)](https://en.wikipedia.org/wiki/Electromagnetic_interference), which can jam or interfere with signals transmissions on the 2.4 GHz radio band frequency, and degrade the wireless communication with your Zigbee adapter/devices.

For example, interference from USB 3.0 ports and unshielded USB peripheral cables are especially infamously known to affect 2.4 GHz radio reception for low-power/low-bandwidth devices. Therefore you should always place your Zigbee adapter far away as possible from any potential sources of EMI/EMI/RMI, preferably by using an adequately long shielded USB extension cable connected to a USB 2.0 port.

Zigbee also uses [mesh networking topology](https://en.wikipedia.org/wiki/Mesh_networking), which means that most mains-powered devices are a "Zigbee Router" that can act as a signal repeater and range extended by transmitting data over long distances by passing data messages through the Zigbee network mesh of intermediate devices to reach more distant Zigbee devices. Thus to have a healthy Zigbee network you need many Zigbee Router devices relatively close to each other in order to achieve good coverage and range.

#### Simple actions that should improve most Zigbee setups and common root causes of interference

- Zigbee adapter hardware:
  - Bad performance from old/outdated/obsolete Zigbee adapter hardware or poor Zigbee adapter antenna:
    - Buy and use a supported Zigbee USB adapter based on newer/modern chip hardware.
      - Consider a Zigbee adapter that has an external antenna.
      - While older adapters might work, they could have obsolete hardware or old firmware, which prevents reliable operation.
  - Poor or outdated Zigbee adapter firmware on the Zigbee adapter:
    - Update to a later Zigbee chip firmware on the Zigbee adapter. Updating firmware is usually straightforward if the manufacturer or the chip maker provides one.
- Zigbee adapters are RFI sensitive and can be very susceptible to all types of EMI/EMF interference:
  - Poor placement of the Zigbee adapter or wrong orientation of the Zigbee adapter antenna:
    - Use a long USB extension cable to place the Zigbee adapter away from interference and obstacles.
      - Ensure the USB extension cable is adequately shielded (thick cables usually have this).
        - A USB extension cable makes orienting the Zigbee adapter/antenna easier.
    - Try different physical placement and orientations of the Zigbee adapter or its antenna:
      - The optimal placement of the Zigbee adapter is close to the middle of the house as possible.
      - Try to place the Zigbee adapter at some distance away from walls, ceilings, and floors.
      - Try different orientations of the adapter's external antenna (or the whole Zigbee adapter).
  - USB 3.0 ports/computers/peripherals are known culprits of RFI/EMI/EMF disruption. (See Ref. [1](https://www.usb.org/sites/default/files/327216.pdf) and [2](https://www.unit3compliance.co.uk/2-4ghz-intra-system-or-self-platform-interference-demonstration/)).
    - Make sure to only connect the Zigbee USB adapter to a USB 2.0 port (and not to a USB 3.x port). 
      - If your computer only has a USB 3.x port then buy and connect the adapter via a powered USB 2.0 hub:
        - Using via USB 2.0 hub will in practice convert USB 3.0 to a USB 2.0 port and thus avoid USB 3.0 EMF.
          - A USB 2.0 hub that uses an external power supply will ensure power requirements are fulfilled.
    - Shield any unshielded computers/peripherals/devices by adding all-metal enclosures/chassis/casings.
      - Single-board-computers and USB 3.x hard drives are especially known as sources of EMF/EMI/RFI.
        - Be aware metal casings can decrease the performance of internal/built-in Zigbee adapters.
      - Also, be sure to use adequately shielded USB cables for any such peripherals/devices too.
  - 2.4 GHz RF Interference (RFI) from Wi-Fi Routers and Wi-Fi Access Points or other devices:
    - First of all, try to place your Zigbee adapter away from Wi-Fi access points or sources of WiFi.
    - Place Zigbee adapters far away from electrical/power wires/cables, power supplies, and household appliances.
    - Zigbee could have overlapping frequency ranges with Wi-Fi, see the section above on defining Zigbee channel use.
  - Add more and decrease the distance between Zigbee devices in Zigbee network mesh to get better range and coverage:
    - Zigbee uses [mesh networking](https://en.wikipedia.org/wiki/Mesh_networking) and depends on having many "Zigbee Router" devices to extend range and coverage:
      - Recommendation is to add additional mains-powered Zigbee devices known to be good Zigbee Router devices.
        - Add more Zigbee Router devices and reduce their distances to extend network mesh coverage and range.
          - Note that not all mains-powered devices have firmware that makes them act as a Zigbee Router device.
          - Some brands/models of Zigbee Router devices are known to only work well with the same brand of devices.
      - Buy a few known good dedicated Zigbee Router products (for example the “[IKEA Tradfri Signal Repeater](https://www.google.com/search?q=IKEA+Tradfri+Signal+Repeater)”).
        - Search community forums for more "Zigbee signal repeater" or "Zigbee range extender" Zigbee Router tips.
      - Buy a few additional new Zigbee USB adapters to use after re-flashing them with Zigbee Router firmware.
        - Reflash/reuse modern Zigbee USB adapters to act as Zigbee Router devices by changing Zigbee firmware.

**Comment by Hedda** [comment]
@frenck https://github.com/home-assistant/home-assistant.io/pull/18864 pull request is still locked so no one including myself can comment there.

![image](https://user-images.githubusercontent.com/6320001/216129808-5a853ffa-1f99-4c7f-88b3-ea6cb51d5e60.png)

**Comment by Hedda** [comment]
FYI, while https://github.com/home-assistant/home-assistant.io/pull/18864  made it into the ZHA integration documentation, please also read/follow/join the directly related community forum discussion here which covers more detailes and solutions -> https://community.home-assistant.io/t/guide-for-zigbee-interference-avoidance-and-network-range-coverage-optimization/515752
